tests/ci/upload-coverage-report.sh

```diff
@@ -1,10 +0,0 @@
-#! /bin/bash
```

Comment:
Removed why?

Reply:
No functional change was made. .travis.yml has all the stuff

Reply:
+1